script/cileaks.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -x
+
+# If this platform doesn't support test execution, bail out now
+if [ -n "$SKIP_TESTS" ];
+then
+	exit $?;
+fi
+
+if [ -n "$VALGRIND" -a -e "$(which valgrind)" ]; then
```

Comment:
I'd say we should error out if we request a Valgrind build without Valgrind being installed. Otherwise, we'll get a green CI even if the job doesn't do what it's supposed to do